source/docs/software/driverstation/driver-station-log-viewer.rst

```diff
@@ -116,4 +116,4 @@ An issue with the ethernet cable on the robot is primarily indicated by the ping
 .. image:: images/driver-station-log-viewer/radioreboot.png
    :alt: The ping here shows the roboRIO and the radio as BAD.
 
-A reboot of the robot radio is typically characterized by a loss of connection to the radio for ~40-45 seconds. In this example, the radio briefly lost power at 3:22:44, causing it to start rebooting. The event at 3:22:45 indicates that the ping to the radio failed. At 3:23:11, the DS regains communication with the roboRIO and the roboRIO indicates it has been up for 1272.775 seconds, ruling out a roboRIO reboot. Note that the network switch on the radio comes back up very quickly so a momentary power loss may not result in a "radio lost"/"radio seen" event pair. A longer disturbance may result in radio events being logged by the DS. In that case, the distinguishing factor which points towards a radio reboot is the ping status of the radio from the DS. If the radio resets, the radio will be unreachable. If the issue is a cabling or connection issue on the robot, the radio ping should remain "GOOD".
+A reboot of the robot radio is typically characterized by a loss of connection to the radio for ~60 seconds. In this example, the radio briefly lost power at 3:22:44, causing it to start rebooting. The event at 3:22:45 indicates that the ping to the radio failed. At 3:23:11, the DS regains communication with the roboRIO and the roboRIO indicates it has been up for 1272.775 seconds, ruling out a roboRIO reboot. Note that the network switch on the radio comes back up very quickly so a momentary power loss may not result in a "radio lost"/"radio seen" event pair. A longer disturbance may result in radio events being logged by the DS. In that case, the distinguishing factor which points towards a radio reboot is the ping status of the radio from the DS. If the radio resets, the radio will be unreachable. If the issue is a cabling or connection issue on the robot, the radio ping should remain "GOOD".
```

Comment:
I thought the radio was supposed to reboot faster then the old radio

Reply:
I asked @kiet to clarify this in our group chat.  I got no answer.  I am basing this off of admittedly old information but I don't know what else to do: https://youtu.be/1VdwLTNM0C8?si=Al0CTTApA_hBWUys&t=247

Reply:
I would go with 40-50 seconds. It is in most cases slightly slower than the OM5P from power on to full link.